MaterialDesignThemes.Wpf/PaletteHelper.cs

```diff
@@ -10,6 +10,15 @@ namespace MaterialDesignThemes.Wpf
 {
     public class PaletteHelper
     {
+        /// <summary>
+        /// The primary swatch
+        /// </summary>
+        public Swatch Primary;
```

Comment:
Do we need this?  I am wary of introducing state.  If we are introducing state then we have to look at this a lot more closely to make sure we are handling it correctly.  Which means scanning resources at startup etc which is a risky scenario for us to confidently say to people "this is your current swatch".

So we should not introduce state.